src/cron/generateInvites.ts

```diff
@@ -0,0 +1,55 @@
+import fastq from 'fastq';
+import { Cron } from './cron';
+import { Feature, FeatureValue, Invite } from '../entity';
+import { generateUUID } from '../ids';
+
+const cron: Cron = {
+  name: 'generate-invites',
+  handler: async (con, logger) => {
```

Comment:
Instead of querying the data and then processing it on the application level, we can run one insert query that will insert data based on select statement.
Look here for reference: https://dba.stackexchange.com/a/2974
This would make this cron much more simple

Reply:
I think the only concern here is how we're going to generate the token as it currently happens on app level. If we can generate on DB level, then I think we can do select insert.

Reply:
I changed this to be generated at the db level, and updated the corn job to run one query for everything. Thanks for the suggestion @idoshamun & @sshanzel